README.md

```diff
@@ -1 +1,73 @@
-# Mobile-version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Please you should combine the middle mobile version  and the start of mobile version projects in one HTML document so you can have your portfolio completed

Reply:
First comment, l have combine both projects.